web/react/stores/post_store.jsx

```diff
@@ -150,6 +150,20 @@ var PostStore = assign({}, EventEmitter.prototype, {
   getDraft: function(channel_id, user_id) {
     return BrowserStore.getItem("draft_" + channel_id + "_" + user_id);
   },
+  storeCommentDraft: function(user_id, parent_post_id, draft) {
```

Comment:
I think it's OK to add functions in that are not being used when it's likely we're going to need them in the future, especially when they're minor ones like this. If it was a big, complicated function or there was a ton of them, then it might be a problem